src/source-af-packet.c

```diff
@@ -1005,8 +1152,13 @@ static int AFPReadAndDiscardFromRing(AFPThreadVars *ptv, struct timeval *synctv)
         return 1;
     }
 
+    /* FIXME specific code to write */
```

Comment:
No, I was going to remove this FIXME and forgot about it. This is just a workaround to avoir to have dropped packets at start. And it seems it does not work very well.